var/spack/repos/builtin/packages/r-hdf5r/package.py

```diff
@@ -20,21 +20,15 @@ class RHdf5r(RPackage):
     url      = "https://cloud.r-project.org/src/contrib/hdf5r_1.2.0.tar.gz"
     list_url = "https://cloud.r-project.org/src/contrib/Archive/hdf5r"
 
+    version('1.3.5',
+            sha256='87b75173ab226a9fbaa5b28289349f3c56b638629560a172994b8f9323c1622f')
     version('1.3.3', sha256='a0f83cbf21563e81dbd1a1bd8379623ed0c9c4df4e094c75013abfd7a5271545')
     version('1.2.0', sha256='58813e334fd3f9040038345a7186e5cb02090898883ac192477a76a5b8b4fe81')
 
     depends_on('r@3.2.2:', type=('build', 'run'))
     depends_on('r-r6', type=('build', 'run'))
     depends_on('r-bit64', type=('build', 'run'))
-    depends_on('hdf5@1.8.13:')
-
-    def configure_args(self):
-        if 'mpi' in self.spec:
-            args = [
-                '--with-hdf5={0}/h5pcc'.format(self.spec['hdf5'].prefix.bin),
-            ]
-        else:
-            args = [
-                '--with-hdf5={0}/h5cc'.format(self.spec['hdf5'].prefix.bin),
-            ]
-        return args
+    depends_on('hdf5@1.8.13:+hl')
+    depends_on('pkgconfig', type='build')
+
+    patch('configure.patch')
```

Comment:
Can you put a comment above the directive explaining briefly what the patch does, why it is needed and (if taken from elsewhere) point at the provenance?

Reply:
I pushed a commit with a comment. Let me know if it is not brief enough.

Reply:
@glennpj Thanks, and sorry for the delay in replying.